### DISCUSSION TITLE
Draggable

### DISCUSSION THREAD

**Discussion by dbrnjh47** [saade/filament-fullcalendar#190] [Q&A]
I still can’t find an example or guess how I could implement a drop.
Can you tell me how I can implement dragging a block from the list to create it in the calendar for a specific day

[demo](https://fullcalendar.io/docs/external-dragging-demo)
[droppable](https://fullcalendar.io/docs/droppable)
[You must also set the calendar’s droppable setting to true and import the interaction plugin](https://fullcalendar.io/docs/external-dragging)

I inserted an example with dragging a block, everything works, but the calendar does not respond at all

![image](https://github.com/saade/filament-fullcalendar/assets/19250635/79e4c35d-0de4-4dd1-a08e-f319c6e0c860)

**Comment by krv** [comment]
Following

**Comment by chosten** [comment]
If you did it like me, you imported your own copy of FullCalendar and then tried to add Draggable elements to the FullCalendar initialized by this package. That does not work. FullCalendar is using some kind of store that is not shared between "your" FullCalendar and the one in this package.
I fixed this by using my own view for the widget and then loading a different JS file from there by changing the `ax-load-src` attribute.

**Reply by krv** [reply]
> I fixed this by using my own view for the widget and then loading a different JS file from there by changing the `ax-load-src` attribute.

Thank you for your suggestion! Would you mind sharing more details about this?

**Reply by chosten** [reply]
I'm not sure what details you need. Just create the widget with his own view and copy the content from the [view](https://github.com/saade/filament-fullcalendar/blob/3.x/resources/views/fullcalendar.blade.php) of this package then customize it. Do the same with the [JS](https://github.com/saade/filament-fullcalendar/blob/3.x/resources/js/filament-fullcalendar.js) file.
